pkg/driver/external/proto/driver.proto

```diff
@@ -0,0 +1,144 @@
+// [WIP]: Please do not review this file yet.
+
+syntax = "proto3";
+
+package driver;
+
+option go_package = "github.com/lima-vm/lima/pkg/driver/external/proto";
+
+import "google/protobuf/empty.proto";
+
+service Driver {
+    rpc Validate(google.protobuf.Empty) returns (ValidateResponse);
+    rpc Initialize(google.protobuf.Empty) returns (InitializeResponse);
+    rpc CreateDisk(google.protobuf.Empty) returns (CreateDiskResponse);
+    rpc Start(google.protobuf.Empty) returns (StartResponse);
+    rpc Stop(google.protobuf.Empty) returns (StopResponse);
+
+    rpc Name(google.protobuf.Empty) returns (NameResponse);
+    rpc SetConfig(SetConfigRequest) returns (SetConfigResponse);
+    
+    rpc GetVSockPort(google.protobuf.Empty) returns (GetVSockPortResponse);
+    rpc GetVirtioPort(google.protobuf.Empty) returns (GetVirtioPortResponse);
+}
+
+message Empty {}
+
+message InstanceConfig {
```

Comment:
Can we just pass the byte array of YAML (or JSON), so that we do not need to sync the YAML struct with this proto definition?